### ISSUE TITLE
result contains null

### ISSUE THREAD

**Issue by jackneer** [nikbelikov/tsp-solver#1]
I passed points with length <= 4
```
const POINTS = [
  { id: 0, name: "Saint Petersburg" },
  { id: 1, name: "Moscow" },
  { id: 2, name: "Pskov" },
  { id: 3, name: "Cherepovets" }  
];
```
the result contains null as following
```
[{"id":0,"name":"Saint Petersburg"},null,null,{"id":3,"name":"Cherepovets"}]
```
I tried locally and online https://codesandbox.io/s/cezb9 both v0.3.2 and got the same result.

**Comment by nikbelikov** [comment]
@jackneer Oh, this is a bug. I will fix it soon. Thank you!

**Comment by nikbelikov** [comment]
@jackneer fixed in 0.4.0 https://codesandbox.io/s/magical-newton-etqts
